necessary.sh

```diff
@@ -0,0 +1,5 @@
+curl -fsSL -o libsodium-src.tar.gz "https://github.com/jedisct1/libsodium/releases/download/1.0.16/libsodium-1.0.16.tar.gz"
+mkdir libsodium-src
+tar -zxf libsodium-src.tar.gz -C libsodium-src --strip-components=1
+cd libsodium-src/
+sudo ./configure &&  sudo make -j$(nproc) &&  sudo make install && sudo ldconfig
```

Comment:
Do we really need to download libsodium sources? Isn't there an ubuntu package with this lib?

Reply:
Also: you shouldn't run `make -j$(nproc)` as root

Reply:
If we really need to download and compile libsodium then maybe we can avoid installing it to system? You already use `LD_LIBRARY_PATH` to some path. This configured properly should be enough.

Reply:
compilation is needed, as repos are outdated

Reply:
but this file is not needed, a note in the readme is sufficient I think

Reply:
I did this in the `Dockerfile`

Reply:
Ok I will make a note in the readme and remove this script